<ui version="4.0" >
 <class>QLocalDatabaseConfigurationScreenBase</class>
 <widget class="QWidget" name="QLocalDatabaseConfigurationScreenBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>395</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>3</hsizetype>
    <vsizetype>5</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize" >
   <size>
    <width>20000</width>
    <height>5000</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Local Database Configuration</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../main/main.qrc" >:/images/database.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" colspan="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>810</width>
       <height>25</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="0" colspan="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>830</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0" colspan="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>810</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="10" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>16</width>
         <height>68</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_13" >
         <property name="text" >
          <string>Local database maintenance:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>810</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QPushButton" name="m_buttonCreateDatabase" >
           <property name="text" >
            <string>Create database</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="m_buttonCompactDatabase" >
           <property name="toolTip" >
            <string>Compact Database</string>
           </property>
           <property name="text" >
            <string>Compact Database</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="m_buttonDeleteStudies" >
           <property name="toolTip" >
            <string>Delete all the studies of the cache</string>
           </property>
           <property name="text" >
            <string>Delete all Studies</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>391</width>
             <height>23</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>31</width>
         <height>8</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_buttonApplyCache" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="toolTip" >
        <string>Apply the canges</string>
       </property>
       <property name="layoutDirection" >
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text" >
        <string>Apply Changes</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="textLabel3_7" >
       <property name="text" >
        <string>Minimum space required in harddisk to retrieve dicom objects:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_textMinimumSpaceRequiredToRetrieve" >
       <property name="maximumSize" >
        <size>
         <width>40</width>
         <height>32767</height>
        </size>
       </property>
       <property name="maxLength" >
        <number>4</number>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="textLabel4_6" >
       <property name="text" >
        <string>Gb</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>378</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QCheckBox" name="m_checkBoxDeletedOldStudies" >
       <property name="text" >
        <string>Delete studies automatically that have not been viewed last</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_textMaximumDaysNotViewed" >
       <property name="maximumSize" >
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maxLength" >
        <number>4</number>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>days</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>121</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QCheckBox" name="m_checkBoxDeleteStudiesIfNotEnoughFreeSpaceAvailable" >
       <property name="text" >
        <string>When there isn't enough free space to retrieve or import new studies delete old studies until get</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_textSpaceToFreeIfNotEnoughSpaceAvailable" >
       <property name="maximumSize" >
        <size>
         <width>35</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maxLength" >
        <number>4</number>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>Gb of free space for starviewer.</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="8" column="0" colspan="2" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="textLabel1_9" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>5</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>90</width>
           <height>0</height>
          </size>
         </property>
         <property name="text" >
          <string>Database path :</string>
         </property>
         <property name="textFormat" >
          <enum>Qt::RichText</enum>
         </property>
         <property name="alignment" >
          <set>Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel1_8" >
         <property name="minimumSize" >
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
         <property name="text" >
          <string>Cache images path :</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="1" >
        <widget class="QPushButton" name="m_buttonExaminateDataBase" >
         <property name="toolTip" >
          <string>Examine disk  to find the database</string>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="icon" >
          <iconset resource="../main/main.qrc" >:/images/folderopen.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLineEdit" name="m_textCacheImagePath" />
       </item>
       <item row="1" column="1" >
        <widget class="QPushButton" name="m_buttonExaminateCacheImage" >
         <property name="toolTip" >
          <string>Examine disk to find the directory to save the retrieved images</string>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="icon" >
          <iconset resource="../main/main.qrc" >:/images/folderopen.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QLineEdit" name="m_textDatabaseRoot" />
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>m_textDatabaseRoot</tabstop>
  <tabstop>m_textCacheImagePath</tabstop>
  <tabstop>m_textMaximumDaysNotViewed</tabstop>
  <tabstop>m_textMinimumSpaceRequiredToRetrieve</tabstop>
  <tabstop>m_buttonCreateDatabase</tabstop>
  <tabstop>m_buttonCompactDatabase</tabstop>
  <tabstop>m_buttonDeleteStudies</tabstop>
  <tabstop>m_buttonApplyCache</tabstop>
  <tabstop>m_buttonExaminateDataBase</tabstop>
  <tabstop>m_buttonExaminateCacheImage</tabstop>
 </tabstops>
 <resources>
  <include location="../main/main.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>m_checkBoxDeleteStudiesIfNotEnoughFreeSpaceAvailable</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_textSpaceToFreeIfNotEnoughSpaceAvailable</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>346</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel" >
     <x>564</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_checkBoxDeletedOldStudies</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_textMaximumDaysNotViewed</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>303</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel" >
     <x>484</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
